.builders/images/linux-x86_64/Dockerfile

```diff
@@ -154,6 +156,29 @@ RUN \
  SHA256="7ccfc6abd01ed67c1a0924b353e526f1b766b21f42d4562ee635a8ebfc5bb38c" \
  RELATIVE_PATH="cyrus-sasl-{{version}}" \
  bash install-from-source.sh --with-dblib=lmdb --enable-gssapi=/usr/local
+# curl
```

Comment:
Should we do this as soon as possible?

Reply:
Why? It's currently only included by the confluent-kafka-python wheel, therefore it made sense to group it with the rest.

Reply:
My point was that we use curl throughout via `.builders/images/install-from-source.sh` so it would be nice to use the same version everywhere.

Reply:
For consistency, I prefer to use the pre-installed curl everywhere, to the point where I'm explicitly avoiding the use of the one installed in this step. I'm only interested in the library, because librdkafka needs it.